portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+document.addEventListener("DOMContentLoaded", function() {
+  const prev = document.getElementById("prev");
+  const next = document.getElementById("next");
+  
+  var currSlide = 1;
```

Comment:
please use let instead of var.

var will be made available everywhere (it escapes the scope). let is only available within this scope.